packages/fx-core/src/component/debugHandler/bot.ts

```diff
@@ -210,13 +210,8 @@ export class BotDebugHandler {
           this.projectSettingsV3.appName,
           MaxLengths.AAD_DISPLAY_NAME
         );
-        const botAuthCredential = await AADRegistration.registerAADAppAndGetSecretByGraph(
-          tokenResult.value,
-          displayName
-        );
-
+        const botAuthCredential = await GraphClient.registerAadApp(tokenResult.value, displayName);
         // set objectId, botId, botPassword to state
-        this.envInfoV3.state[ComponentNames.TeamsBot].objectId = botAuthCredential.objectId;
```

Comment:
`objectId` is no need now? If so, can also remove it from the last line comment, `// set  botId, botPassword to state`.

Reply:
Yes, will update it.